src/Files.App/Actions/Open/OpenContextMenuAction.cs

```diff
@@ -0,0 +1,96 @@
+﻿using CommunityToolkit.WinUI.UI;
+using Files.App.Commands;
+using Files.App.Extensions;
+using Files.App.Filesystem;
+using Files.App.UserControls;
+using Files.App.UserControls.Widgets;
+using Files.App.ViewModels.Widgets;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using Microsoft.UI.Xaml.Input;
+using System.Threading.Tasks;
+using Windows.Foundation;
+using Windows.System;
+
+namespace Files.App.Actions
+{
+	internal class OpenContextMenuAction : IAction
+	{
+		public string Label { get; } = "OpenContextMenu".GetLocalizedResource();
+		public string Description => "TODO: Need to be described.";
+
+		public HotKey HotKey { get; } = new(VirtualKey.Enter, VirtualKeyModifiers.Menu);
```

Comment:
Can you remove the enter key?

Reply:
> Perhaps you mean the key that displays the menu

This is what I meant, and when we add support for remapping you'll be able to switch to Ctrl+Enter but the default should be the regular menu/application key.

Reply:
This key is not present on all keyboards. Why not use the 2 default hotkeys ?

Reply:
Do you know of any other apps using Ctrl+Enter for this?

Reply:
None. This is an opportunity to stand out.

Reply:
Can you add it to the list of shortcuts on the website?

Reply:
I don't know enough node.js to do this quickly. Surely someone else can do this. Currently I am preparing a pr to modify hotkeys and store custom hotkeys in a litedatabase.

Reply:
Actually you have only to edit a markdown file (See https://github.com/files-community/Website/pull/413 for reference)

Reply:
> Currently I am preparing a pr to modify hotkeys and store custom hotkeys in a litedatabase.

Wouldn't json be better?

Reply:
This is out of scope for this pull request, the main thing is to update the file on the website documenation to include the shortcut.

Reply:
I put the shortcuts Ctrl+Enter and the Application key.